shared/reducers/tracker2.tsx

```diff
@@ -114,13 +111,15 @@ export default Container.makeReducer<Actions, Types.State>(initialState, {
   [EngineGen.keybase1NotifyTrackingNotifyUserBlocked]: (draftState, action) => {
     const {blocker, blocked} = action.payload.params.b
     const d = getDetails(draftState, blocker)
-    const followers = d.followers ?? new Set()
-    d.followers = followers
-    const toProcess = blocked ?? []
-    toProcess.forEach(e => {
-      followers.delete(e)
-      getDetails(draftState, e).blocked = true
+    const blockees = blocked ?? []
+    const toProcess = blockees.map<[string, Types.Details]>(username => [
```

Comment:
this explicit type likely not needed

Reply:
without it it turned into `Array<string | Types.Details>`